sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -414,7 +414,7 @@ def test_lasso_lars_ic():
     rng = np.random.RandomState(42)
     X = diabetes.data
     y = diabetes.target
-    X = np.c_[X, rng.randn(X.shape[0], 4)]  # add 4 bad features
+    X = np.c_[X, rng.randn(X.shape[0], 5)]  # add 4 bad features
```

Comment:
I need this to have enough alpha on the grid so alpha_bic > alpha_aic (otherwise it was equal)